src/screens/PairDevices/Scanning.tsx

```diff
@@ -32,11 +33,17 @@ export default function Scanning({ onTimeout, onError, onSelect }: Props) {
   const renderItem = useCallback(
     ({ item }) => {
       const knownDevice = knownDevices.find(d => d.id === item.id);
+      let modelId = "nanoX" as DeviceModelId;
+      if (item?.serviceUUIDs.length) {
```

Comment:
Shouldn't you add a `?` optionnal chaining on length ? If `serviceUUIDs` is undefined, should be problematic to get the length, isn't it ?